### PR TITLE
PMM-11113: new Portal connection layout

### PR DESCRIPTION
**Pull request by fabio-silva**
**What this PR does / why we need it**: Changes the layout for the Portal connection from PMM

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-11113

Screenshot 2023-01-27 at 14 48 22

### CODE REVIEW

File: public/app/percona/settings/components/Platform/Connect/Connect.tsx [hunk 1]
```diff
@@ -16,48 +16,73 @@ import { getStyles } from './Connect.styles';
...
+      </p>
+      <a href={`${saasHost}/login`} rel="noreferrer noopener" target="_blank">
+        <Button variant="secondary" icon="external-link-alt">
+          {Messages.createPerconaAccountAnchor}
+        </Button>
+      </a>
+      <h2 className={styles.connectionTitle}>{Messages.connectTitle}</h2>
+      <form data-testid="connect-form" className={styles.form} onSubmit={handleSubmit} autoComplete="off">
+        <div className={styles.serverDetails}>
+          <TextInputField name="pmmServerId" disabled label={Messages.pmmServerId} fieldClassName="my-class" />
```
Comment (matejkubinec) [inline public/app/percona/settings/components/Platform/Connect/Connect.tsx:46]: What does the `fieldClassName="my-class"` mean here?

File: public/app/percona/settings/components/Platform/Connect/Connect.tsx [hunk 2]
```diff
@@ -16,48 +16,73 @@ import { getStyles } from './Connect.styles';
...
+            required
+            disabled={connecting}
+          />
+        </div>
+        <div className={styles.accessTokenRow}>
+          <TextInputField
+            name="accessToken"
+            label={Messages.accessToken}
+            validators={[validators.required]}
+            placeholder="Token here..."
```
Comment (matejkubinec) [inline public/app/percona/settings/components/Platform/Connect/Connect.tsx:61]: Placeholder could be added to messages.

File: public/app/percona/settings/components/Platform/Connect/Connect.tsx [hunk 3]
```diff
@@ -16,48 +16,73 @@ import { getStyles } from './Connect.styles';
...
+            showErrorOnBlur
+            required
+            disabled={connecting}
+          />
+          <a href={`${saasHost}/profile`} rel="noreferrer noopener" target="_blank" className={styles.getTokenAnchor}>
+            <Button variant="secondary" fill="text" icon="external-link-alt">
+              Get token
```
Comment (matejkubinec) [inline public/app/percona/settings/components/Platform/Connect/Connect.tsx:68]: Button text could be added to messages.

**Review by matejkubinec — CHANGES_REQUESTED** [review]
Just really small nitpicks, otherwise looks great!

**Review by matejkubinec — DISMISSED** [review]

**Review by filipmikes1 — APPROVED** [review]

**Pull request merged**